src/css/bs-stepper.css

```diff
@@ -1,198 +1,68 @@
-/*!
- * bsStepper v1.3.0 (https://github.com/Johann-S/bs-stepper)
- * Copyright 2018 Johann-S <johann.servoire@gmail.com>
- * Licensed under MIT (https://github.com/Johann-S/bs-stepper/blob/master/LICENSE)
- */
-
-.bs-stepper {
-  display: block;
-}
-
-.bs-stepper.vertical {
-  display: flex;
-}
-
-.bs-stepper .bs-stepper-header {
+.bs-stepper-btn {
+  padding: 24px;
+  font-weight: 700;
+  text-align: center;
+  text-decoration: none;
```

Comment:
I don't understand why did you add this new CSS class ? `.bs-stepper-btn`

Reply:
It's better to add specific classes if you want to theme an element.